figures/line.py

```diff
@@ -21,10 +22,20 @@ def _get_line(self, data, key):
             y_axis_label="No. Violations",
         )
         p.line(x, y, legend_label="Progress", line_width=2)
+        try:
+            p.y_range = Range1d(0, (int(max(y)) * 1.1))
+        except ValueError:
+            logging.warning(("No data to plot created for" + key))
+            return
         return p
 
     def get_figure(self):
-        result = [self._get_line(self._data, key) for key in self._data]
+        result = []
+        for key in self._data:
+            output = self._get_line(self._data, key)
+            if output is None:
+                continue
```

Comment:
A small comment here describing when and why output is none - so it makes sense why we just continue over a None :)

Reply:
roger!